pkg/pb/gracefulexit.proto

```diff
@@ -55,10 +55,11 @@ message TransferSucceeded {
   metainfo.AddressedOrderLimit addressed_order_limit   = 1;
   orders.PieceHash             original_piece_hash     = 2;
   orders.PieceHash             replacement_piece_hash  = 3;
+  bytes piece_id    = 4 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false]; // the original piece ID  
```

Comment:
Rename the field to `original_piece_id` rather than using comments for it.

Reply:
Updated